funcx_endpoint/funcx_endpoint/tests/test_manager.py

```diff
@@ -0,0 +1,54 @@
+from funcx_endpoint.endpoint.manager import EndpointManager
+import os
+import logging
+import sys
+import shutil
+
+logger = logging.getLogger('mock_funcx')
+
+def test_configure():
+    manager = EndpointManager(logger)
+    manager.FUNCX_DIR = f'{os.getcwd()}'
+    config_dir = os.path.join(manager.FUNCX_DIR, "mock_endpoint")
+
+    try:
+        manager.configure_endpoint("mock_endpoint", None)
+        if os.path.exists(config_dir):
+            shutil.rmtree(config_dir)
+            print('\nSuccessful: config dir created and removed.')
+    except:
+        raise Exception('Config dir has not been generated as expected.')
+
+
+def test_start():
+    manager = EndpointManager(logger)
+    manager.FUNCX_DIR = f'{os.getcwd()}'
+    config_dir = os.path.join(manager.FUNCX_DIR, "mock_endpoint")
+
+    try:
+        manager.configure_endpoint("mock_endpoint", None)
+        manager.start_endpoint("mock_endpoint", None)
```

Comment:
Need to mock the things that are called by this method:
* zmq.auth.create_certificates
* FuncXClient

Then [verify](https://www.kite.com/python/answers/how-to-assert-that-a-method-was-called-in-a-unit-test-in-python) that they are called with the expected arguments